mm2src/coins/utxo.rs

```diff
@@ -1318,6 +1360,31 @@ pub fn utxo_coin_from_iguana_info(
     Ok(UtxoCoin(Arc::new(coin)).into())
 }
 
+/// Function calculating KMD interest
+/// https://komodoplatform.atlassian.net/wiki/spaces/KPSD/pages/71729215/What+is+the+5+Komodo+Stake+Reward
+/// https://github.com/KomodoPlatform/komodo/blob/master/src/komodo_interest.h
+fn kmd_interest(height: u64, value: u64, lock_time: u64, current_time: u64) -> u64 {
+    const KOMODO_ENDOFERA: u64 = 7777777;
+    const LOCKTIME_THRESHOLD: u64 = 500000000;
+    // interest will stop accrue after block 7777777
+    if height >= KOMODO_ENDOFERA { return 0 };
+    // interest doesn't accrue for lock_time < 500000000
+    if lock_time < LOCKTIME_THRESHOLD { return 0; }
+    // current time must be greater than tx lock_time
+    if current_time < lock_time { return 0; }
+
+    let mut minutes = (current_time - lock_time) / 60;
+    // at least 1 hour should pass
+    if minutes < 60 { return 0; }
+    // interest stop accruing after 1 year before block 1000000
+    if minutes > 365 * 24 * 60 { minutes = 365 * 24 * 60 };
```

Comment:
@artemii235 , the number of minutes in a day is 86400 / 60 = 1440. I wonder if `365 * 24 * 60`
 should be `365 * 24 * 1440`?
Thanks for the comments.